### PR TITLE
waybar: 0.4.0 -> 0.5.0

### PR DESCRIPTION
**Pull request by minijackson**
###### Motivation for this change

Bigger version numbers are better.
https://github.com/Alexays/Waybar/releases/tag/0.5.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/waybar/default.nix [hunk 1]
```diff
@@ -3,17 +3,18 @@
 , traySupport  ? true,  libdbusmenu-gtk3
 , pulseSupport ? false, libpulseaudio
 , nlSupport    ? true,  libnl
+, udevSupport  ? true,  libudev
```
Comment (dywedir) [inline pkgs/applications/misc/waybar/default.nix]:
```suggestion
, udevSupport  ? true,  udev
```

File: pkgs/applications/misc/waybar/default.nix [hunk 2]
```diff
@@ -25,19 +26,21 @@
       ++ optional  traySupport  libdbusmenu-gtk3
       ++ optional  pulseSupport libpulseaudio
       ++ optional  nlSupport    libnl
+      ++ optional  udevSupport  libudev
```
Comment (dywedir) [inline pkgs/applications/misc/waybar/default.nix]:
```suggestion
      ++ optional  udevSupport  udev
```

**Comment by minijackson** [comment]
There you go ^^

**Comment by dywedir** [comment]
@GrahamcOfBorg build waybar

**Comment by dywedir** [comment]
Thanks!

**Pull request merged**